<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Cascadia Code SemiBold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #97afcc;
color:#e6eefc;</string>
   </property>
   <widget class="QPushButton" name="AddStudent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>Add student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeeFullList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>See full list</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="StudentView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>721</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
    </property>
    <column>
     <property name="text">
      <string>Surname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cathedra</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Group</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Average point</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Semester</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="ShellSort">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Semester</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShellEnterYear">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShellEnterSem">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #46298a;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RangeMarkBegin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RangeMarkEnd">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Range of scores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadList">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>Read from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WriteList">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>Write in file</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,2,2,2,2,2,2,2,2,2,1,2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,3,1">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="BestRanking">
         <property name="maximumSize">
          <size>
           <width>777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid; 
border-color: #457cba;
border-radius:5px;
background-color: #457cba;
color:#d5e1f5;</string>
         </property>
         <property name="text">
          <string>Success grading</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Best year:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BestYear">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Worst year:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="WorstYear">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Cascadia Code SemiBold</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Top 3 students:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,6">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="gold_img">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Top1">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,6">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="silver_img">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Top2">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,6">
       <item>
        <widget class="QLabel" name="bronze_img">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Top3">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Cascadia Code SemiBold</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>The most succesfull group of year</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="2,3,5">
       <item>
        <widget class="QLineEdit" name="EnterYearGroup">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid; 
border-color: #ebf0f5;
border-radius:10px;
background-color: #ebf0f5;
color:#434445;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BestGroup">
         <property name="font">
          <font>
           <family>Cascadia Code SemiBold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid; 
border-color: #457cba;
border-radius:5px;
background-color: #457cba;
color:#d5e1f5;</string>
         </property>
         <property name="text">
          <string>Determine</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ShowBestGroup">
       <property name="font">
        <font>
         <family>Cascadia Code SemiBold</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ClearList">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code SemiBold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid; 
border-color: #6698d1;
border-radius:5px;
background-color: #6698d1;
color:#d5e1f5;</string>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Cascadia Code SemiBold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
